easybuild/easyconfigs/p/Pillow/Pillow-4.3.0-foss-2017b-Python-3.6.3.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'PythonPackage'
+
+name = 'Pillow'
+version = '4.3.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://pillow.readthedocs.org/'
+description = """Pillow is the 'friendly PIL fork' by Alex Clark and Contributors.
+ PIL is the Python Imaging Library by Fredrik Lundh and Contributors."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['a97c715d44efd5b4aa8d739b8fad88b93ed79f1b33fc2822d5802043f3b1b527']
+
+dependencies = [
+    ('Python', '3.6.3'),
+    ('libjpeg-turbo', '1.5.2'),
+    ('libpng', '1.6.32'),
+    ('zlib', '1.2.11'),
+    ('LibTIFF', '4.0.9'),
+    ('freetype', '2.8.1'),
```

Comment:
@schiotz You'll need to stick to `freetype` 2.8 here to avoid conflicts.

Reply:
@boegel Fixed.  Thanks for pointing it out.